templates/addons/svc/s3-policy.yml

```diff
@@ -1,3 +1,9 @@
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks:
+        - W2001 # Parameter not used
```

Comment:
We could remove the unused `Parameters`, but @lgarvey and something I saw [here](https://github.com/aws/copilot-cli/issues/4730#issuecomment-1595029932) suggest they are needed by Copilot for something. Is it for the tags in AWS?